pkg/webhook/core.oam.dev/v1beta1/application/validation.go

```diff
@@ -107,10 +108,23 @@ func (h *ValidatingHandler) ValidateComponents(ctx context.Context, app *v1beta1
 	return componentErrs
 }
 
+// ValidateAnnotations validates whether the application has both autoupdate and publish version annotations
+func (h *ValidatingHandler) ValidateAnnotations(_ context.Context, app *v1beta1.Application) field.ErrorList {
```

Comment:
can we make this function private? and can we pass annotations only instead of passing app object?